README.md

```diff
@@ -24,11 +24,27 @@ Details on the columns can be found in the [Full samplesheet](docs/usage.md#full
 
 The main parameters are `--input` as defined above and `--output` for specifying the output results directory. You may wish to provide `-profile singularity` to specify the use of singularity containers and `-r [branch]` to specify which GitHub branch you would like to run.
 
-## Profile dists
+## Distance Method and Thresholds
+
+Profile_Dists and the Genomic Address Service workflows can use two distance methods: hamming or scaled.
+
+### Hamming Distances
+
+Hamming distances are integers representing the number of differing loci between two sequences and will range between [0, n], where `n` is the total number of loci. When using Hamming distances, you must specify `--pd_distm hamming` and provide Hamming distance thresholds as integers between [0, n]: `--gm_thresholds "10,5,0"` (10, 5, and 0 loci).
+
+### Scaled Distances
+
+Scaled distances are floats representing the percentage of differing loci between two sequences and will range between [0.0, 100.0]. When using scaled distances, you must specify `--pd_distm scaled` and provide percentages between [0.0, 100.0] as thresholds: `--gm_thresholds "50,20,0"` (50%, 20%, and 0% of loci).
+
+### Thresholds
+
+The `--gm_thresholds` parameter is used to set thresholds for each cluster level, which in turn are used to assign cluster addresses at each level. When specifying `--pd_distm hamming` and `--gm_thresholds "10,5,0"`, all sequences that have no more than 10 loci differences will be assigned the same cluster code for the first level, no more than 5 for the second level, and only sequences that have no loci differences will be assigned the same cluster code for the third level.
```

Comment:
Could you maybe change the wording here to indicate that guarantees on pairwise distances for identical cluster codes at a level depend on the linkage method (the `--gm_method` parameter). There's only a guarantee if using complete linkage clustering.

Reply:
What do you think of this blurb: [d2e2c80](https://github.com/phac-nml/gasnomenclature/pull/16/commits/d2e2c801f60a9a32d974d4d5e2a6af7164a508ac)